doc/install/containers.rst

```diff
@@ -3,6 +3,20 @@
 Ceph Container Images
 =====================
 
+.. important::
+
+   Using the ``: latest`` tag is discouraged. If you use the ``: latest`` 
+   tag, there is no guarantee that the same image will be on each of 
+   your hosts.  Under these conditions, upgrades might not work 
+   properly. Remember that ``: latest`` is a relative tag, and a moving
+   target.
+
+   Instead of the ``: latest`` tag, use explicit tags or image ids. For
+   example:
+
+   ``docker pull ceph/ceph:v15.2.0``
```

Comment:
> Maybe we should also add an example of an explicit tag?
> 
> ```
> docker pull ceph/ceph:v15.2.0
> ```

oops! podman instead of docker would probably be better ..

```
podman pull ceph/ceph:v15.2.0
```

Reply:
I've made this change:

s/docker/podman/